modules/ROOT/pages/environments.adoc

```diff
@@ -54,10 +62,38 @@ To view the client ID and secret for your environment:
 
 . Log in to the Anypoint Platform as an administrator.
 . Go to Access Management.
-. From the left-hand menu, click *Environments*.
+. In the menu on the left, click *Environments*.
 . Click the name of the environment whose client ID/secret you want to view.
 
 [NOTE]
 --
 To change the client ID or client secret of an environment, you must contact your customer support representative.
 --
+
+[[add-an-idp-to-an-existing-environment]]
+== Add a Client Provider to an Existing Environment
+
+If your organization administrator provides multiple xref:managing-api-clients.adoc[client providers] for your organization, you can add them to existing environments.
+
+To add a client provider to an existing environment:
+
+. Log in to the Anypoint Platform as an administrator.
+. Go to Access Management.
+. In the menu on the left, click *Environments*.
+. Click the name of the environment.
+. In the *Client Provider* field, select at least one client provider.
+. If you added more than one client provider, click *Set as default* to designate a default client provider.
+. Click *Update*.
+
+[[remove-an-idp-from-an-environment]]
+== Remove a Client Provider from an Environment
+
+You can remove a client provider from an environment at any time.
+When you remove your default client provider from an environment that has more than one client provider, the next client provider is automatically named as your new default client provider.
+When you remove all external client providers from an environment, the Anypoint Platform client provider becomes the default client provider for that environment.
```

Comment:
Can we say "the built-in Anypoint Platform client provider"

Reply:
```suggestion
When you remove all external client providers from an environment, the built-in Anypoint Platform client provider becomes the default client provider for that environment.
```